xbmc/cores/AudioEngine/Sinks/AESinkAUDIOTRACK.cpp

```diff
@@ -1073,6 +1080,7 @@ void CAESinkAUDIOTRACK::UpdateAvailablePassthroughCapabilities(bool isRaw)
   m_info.m_wantsIECPassthrough = false;
   m_info.m_dataFormats.push_back(AE_FMT_RAW);
   m_info.m_streamTypes.clear();
+  bool supports_192khz = m_sink_sampleRates.find(192000) != m_sink_sampleRates.end();
```

Comment:
```suggestion
  const bool supports_192khz = m_sink_sampleRates.find(192000) != m_sink_sampleRates.end();
```

And maybe add blank line after

Reply:
Thanks - added it.